model/aws_credentials.go

```diff
@@ -28,6 +30,21 @@ type AWSCredentials struct {
 	DevicePolicyName *string        `json:"device_policy_name,omitempty" bson:"device_policydevice_policy_name,omitempty"`
 }
 
+func (c AWSCredentials) MarshalJSON() ([]byte, error) {
```

Comment:
Do you think it could be better to add the `json.Marshaller` (or `encoding.TextMarshaller`) interface to the `crypto.String` type directly?
This way we avoid accidentally leaking any encrypted strings to the API.

Reply:
@alfrunes if you prefer, I can do it that way.